src/python/pants/backend/java/dependency_inference/rules.py

```diff
@@ -48,7 +49,7 @@ class JavaInferredDependencies:
 
 @dataclass(frozen=True)
 class JavaInferredDependenciesAndExportsRequest:
-    address: Address
+    sources_field: SourcesField
```

Comment:
Other part of the Pants code base call this `source` or `sources` depending on the context (i.e., without the `_field` part). Maybe make consistent?